ios/FluentUI/Command Bar/CommandBarAppearance.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+public struct CommandBarButtonAppearance {
+    public var tintColor: UIColor
+    public var backgroundColor: UIColor
+    public var highlightedTintColor: UIColor
+    public var highlightedBackgroundColor: UIColor
+
+    public init(
+        tintColor: UIColor = Defaults.tintColor,
+        backgroundColor: UIColor = Defaults.backgroundColor,
+        highlightedTintColor: UIColor = Defaults.highlightedTintColor,
+        highlightedBackgroundColor: UIColor = Defaults.highlightedBackgroundColor
+    ) {
+        self.tintColor = tintColor
+        self.backgroundColor = backgroundColor
+        self.highlightedTintColor = highlightedTintColor
+        self.highlightedBackgroundColor = highlightedBackgroundColor
+    }
+
+    public enum Defaults {
+        public static let tintColor: UIColor = Colors.gray400
+        public static let backgroundColor: UIColor = Colors.surfaceSecondary
+        public static let highlightedTintColor = UIColor(light: Colors.communicationBlue, dark: .black)
```

Comment:
this probably needs to be MSFColors.primary because not all apps will be communication blue but should their their branding color

Reply:
same for other references